Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM praekeltfoundation/django-bootstrap:py3.6-stretch
+
+WORKDIR /casepro
+
+RUN apt-get-install.sh curl && curl -sL https://deb.nodesource.com/setup_12.x | bash && \
+    apt-get install nodejs -y && apt-get remove curl -y
```

Comment:
There's an `apt-get-purge.sh` script which may be better for removing everything? But that's a minor point